src/api/.rubocop.yml

```diff
@@ -35,6 +35,7 @@ Style/UnneededInterpolation:
 Metrics/BlockLength:
   Exclude:
     - 'config/routes.rb'
+    - 'spec/**/*'
```

Comment:
I think a comment with the reason would be nice (just that this is because the `describe`)